src/picongpu/include/simulation_defines/param/bremsstrahlung.param

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2015 Heiko Burau
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+
+#pragma once
+
+namespace picongpu
+{
+namespace particles
+{
+namespace bremsstrahlung
+{
+
+// Above this value (to the power of three, see comments on mapping) the synchrotron functions are nearly zero.
+BOOST_CONSTEXPR_OR_CONST float_64 SYNC_FUNCS_CUTOFF = 5.0;
+
+// stepwidth for the numerical integration of the bessel function
+BOOST_CONSTEXPR_OR_CONST float_64 SYNC_FUNCS_BESSEL_INTEGRAL_STEPWIDTH = 1e-3;
+
+// Number of sampling points of the lookup table
+BOOST_CONSTEXPR_OR_CONST int SYNC_FUNCS_NUM_SAMPLES = 8192;
```

Comment:
please use an unsigned var, since a number of samples can not be negative.

also be more specific, e.g., `uint32_t`